device-types/Avocent/ACS8016.yaml

```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: Avocent
+model: ACS 8016
+slug: acs8016
+part_number: ACS-8016
+u_height: 1
+is_full_depth: false
```

Comment:
Please add a comment in markdown format if you have a link to the manufacturer provided datasheet